index.html

```diff
@@ -0,0 +1,69 @@
+        <p>The Five Fastest Men in history!</p>
+        <p>3,396,228 views . Jun 6, 2015</p>
+    </section>
+    <aside id="watching-youtube" class="youtuber">
+        <ul class="watching-items">
+            <li>
+                <a class="watching-item" href="#">All</a>
+            </li>
+            <li>
+                <a href="#">Related</a>
+            </li>
+            <li>
+                <a href="#">Watched</a>
+            </li>
+            <li>
+                <a href="#">Recently Watched</a>
+            </li>
+        </ul>
+        <iframe width="640" height="480" src="https://www.youtube.com/embed/0PH0SV4j1Es" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+        <ul>
+            <li>Usain Bolt new 100m world record: 9.58!!!</li>
+        </ul>
+        <iframe width="640" height="480" src="https://www.youtube.com/embed/uJrUXSZE8vA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+        <ul>
+            <li>Usain Bolt Tv docunentary</li>
+        </ul>
+        <iframe width="853" height="480" src="https://www.youtube.com/embed/mP1D-BxVDOE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+        <ul>
+            <li>Usain Bolt's Greatest Hits</li>
+        </ul>
+        <iframe width="853" height="480" src="https://www.youtube.com/embed/88vaihR8DnI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+        <ul>
+            <li>World Record | Men's 200m Final | World Athletics Championships Berlin 2009</li>
+        </ul>
+        <iframe width="853" height="480" src="https://www.youtube.com/embed/uwLDpcye-VM" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+        <ul>
+            <li>Jamaica Break Men's 4x100m World Record - London 2012 Olympics</li>
+        </ul>
+        <iframe width="853" height="480" src="https://www.youtube.com/embed/FuiJHJz4f5Q" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+        <ul>
+            <li>Usain Bolt | ALL Olympic finals + Bonus round | Top Moments</li>
+        </ul>
+    </aside>
+    <footer class="footer">
+
+    </footer>
```

Comment:
Please try to avoid empty elements because they make the code look disorganised.